package.json

```diff
@@ -56,6 +57,7 @@
     "sucrase": "^3.10.1",
     "svelte": "^3.6.9",
     "svelte-loader": "^2.13.6",
+    "typescript": "^3.9.5",
```

Comment:
why does typescript get installed here?

Reply:
I've removed it. I think it was left over from an earlier iteration of working on this. I'm kind of surprised it's not there though because I think it'd be needed to specify which version is being used